install-oracle.sh

```diff
@@ -158,15 +158,12 @@ ORA_RECO_DISKGROUP="${ORA_RECO_DISKGROUP:-RECO}"
 ORA_RECO_DISKGROUP_PARAM="^[a-zA-Z0-9]+$"
 
 ORA_ASM_DISKS="${ORA_ASM_DISKS:-asm_disk_config.json}"
-ORA_ASM_DISKS_PARAM="^.*\.json$"
```

Comment:
I'm not sure we want to lose the parameter validation entirely here.  If the intent is to permit either a blank response or a ".json", something like (^$|\.json$) might work

Reply:
I changed it to "^.+\.json$" - if no value provided default json file name taken otherwise it should be file with json extension